addons/account/models/account_journal_dashboard.py

```diff
@@ -682,15 +682,15 @@ def open_action(self):
 
         # Override the domain only if the action was not explicitly specified in order to keep the
         # original action domain.
+        if action['domain'] and isinstance(action.get("domain"), str):
```

Comment:
```suggestion
        if action.get('domain') and isinstance(action['domain'], str):
```
If `domain` is not defined, you will raise a `KeyError`, so you should use `.get()` in the first part of the `if` statement.
If the first part passes, you know you can access `action['domain']` safely :wink:

Reply:
thanks @epictete ,
updated